app/views/smart_proxies/logs/_list.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= render :partial => "smart_proxies/logs/modal" %>
+<%
+  color_map = {
+    'DEBUG' => 'success',
+    'INFO' => 'info',
+    'WARN' => 'warning',
+    'ERROR' => 'danger',
+    'FATAL' => 'danger'
+  }
+  filter_map = {
+    'DEBUG' => 'I',
+    'INFO' => 'I',
+    'WARN' => 'W',
+    'ERROR' => 'E',
+    'FATAL' => 'E'
+  }
+%>
+<div class="container">
+  <table id="table-proxy-status-logs" class='<%= table_css_classes %> table-fixed'>
+    <thead>
+        <tr>
+            <th class="ca col-md-1 hidden-xs nbsp">Time</th>
+            <th class="ca col-md-1 hidden-sm hidden-xs nbsp">Level
+              <a href="#" onclick="toggleLogsTableFilter(null);" data-toggle="tooltip" data-placement="top" title="Filter ERROR -> WARNING -> INFO/DEBUG -> All"><span class="fa fa-filter"></span></a>
+            </th>
+            <th class="col-md-10 col-sm-11 col-xs-12 nbsp">Message</th>
+            <th class="hidden" id="logs-current-filter">W</th>
+        </tr>
+    </thead>
+    <tbody>
+      <% @smart_proxy.statuses[:logs].log_entries.each do |log| %>
```

Comment:
I would suggest passing `:locals => {:log_entries => @smart_proxy.statuses[:logs].log_entries}` from the controller.